daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md

```diff
@@ -0,0 +1,624 @@
+---
+type: docs
+title: "Quickstart: State Management"
+linkTitle: "State Management"
+weight: 70
+description: "Get started with Dapr's State Store"
+---
+
+Let's take a look at Dapr's [State Management building block]({{< ref state-management >}}). In this quickstart, you will learn how to store and query data as key/value pairs in the [supported state stores]({{< ref supported-state-stores.md >}}).
+
+State management is one of the most common needs of any application: new or legacy, monolith or microservice. Dealing with different databases libraries, testing them, handling retries and faults can be time consuming and hard.
+
+Dapr provides state management capabilities that include consistency and concurrency options. In this guide we’ll start of with the basics: Using the key/value state API to allow an application to save, get and delete state.
+
+<img src="/images/state-management-overview.png" width=800 style="padding-bottom:15px;">
+
+Select your preferred language-specific Dapr SDK before proceeding with the quickstart.
+
+{{< tabs "Python" "JavaScript" ".NET" "Java" "Go" >}}
+ <!-- Python -->
+{{% codetab %}}
+
+### Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+### Step 1: Set up the environment
+
+Clone the sample we've provided.
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
```

Comment:
One small ask.  Can we add a note under here or on line 33 with a link pointing to the github repo itself?  I think it's inconvenient that all of our quickstarts only let you see code if you first clone it.